pkg/pillar/cmd/client/client.go

```diff
@@ -615,8 +730,25 @@ func handleDNSModify(ctxArg interface{}, key string, statusArg interface{}) {
 	}
 
 	// update proxy certs if configured
-	if ctx.zedcloudCtx != nil && ctx.zedcloudCtx.V2API {
-		zedcloud.UpdateTLSProxyCerts(ctx.zedcloudCtx)
+	ctx.zedcloudCtx.DeviceNetworkStatus = &status
+	if ctx.zedcloudCtx != nil {
```

Comment:
You accessed this pointer on the line above; if you need to check for nil you need to do it before that line.
But perhaps you don't need to check for nil?

Reply:
will remove the check.